packages/integrations/src/reportingobserver.ts

```diff
@@ -1,38 +1,30 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 import { EventProcessor, Hub, Integration } from '@sentry/types';
 import { getGlobalObject, supportsReportingObserver } from '@sentry/utils';
 
-/** JSDoc */
 interface Report {
-  [key: string]: any;
+  [key: string]: unknown;
   type: ReportTypes;
   url: string;
   body?: ReportBody;
 }
 
-/** JSDoc */
-const enum ReportTypes {
-  /** JSDoc */
+// We need to export this so users can use the `types` option in the constructor.
+export const enum ReportTypes {
```

Comment:
`const enums` are inlined at use sites by the TS compiler, so this won't work as expected if the export crosses package boundaries.

We'll need to turn it into a regular enum if it's meant to be used by external consumers.